Algorithms/String-Matching/NAIVE_STRING_SEARCHING.cpp

```diff
@@ -0,0 +1,68 @@
+// Program to find all occurrences of a 
+// pattern string within a text string
+// using Naive String Matching Algorithm.
+
+// Author: Pritam Negi.
+
+#include<iostream>
+#include<string>
+using namespace std;
+
+// Function to find the pattern string within the text string
+void NaiveStringMatching(string text, string pattern)
+{
+	// Getting the length of pattern string
+	int pattern_length = pattern.length();
+
+	// Getting the length of text string
+    int text_length = text.length();
+ 	int flag = 0;
+    
+    // Loop to trace the complete text string
+    for (int i = 0; i <= text_length - pattern_length; i++)
+    {
+        int j;
```

Comment:
give i and j proper variable name.
It increases readability.

Reply:
What should I name them instead??